tests/test_cli_compile.py

```diff
@@ -3452,7 +3480,7 @@ def test_compile_recursive_extras_static(runner, tmp_path, current_resolver):
             "--extra",
             "dev",
             "--find-links",
-            os.fspath(MINIMAL_WHEELS_PATH),
+            minimal_wheels_path.as_posix(),
```

Comment:
what's the difference w/ `os.fspath()`?

cc @sirosen

Reply:
`as_posix` uses forward slashes, no matter the platform. os.fspath would use escaped backslashes on windows.

```python
os.fspath(Path.cwd())  # 'C:\\Users\\Administrator\\Downloads'
Path.cwd().as_posix()  # 'C/Users/Administrator/Downloads'
```

This is more of an aesthetic thing, not functional